environments/development.env.template

```diff
@@ -3,3 +3,4 @@ OCAML_BENCH_DB_PASSWORD=docker
 OCAML_BENCH_GRAPHQL_KEY=secret
 OCAML_BENCH_TARGET_ARCH=amd64
 VITE_GRAPHQL_URL=backend_url
+VITE_FRONTEND_URL=frontend_url
```

Comment:
I find the name a bit confusing as "frontend" for me refers to the react web application. That's also the naming we use in the `docker-compose.yaml` file. If I'm correct, this one refers to the pipeline's frontend. If so, wouldn't that be better to use something like `VITE_PIPELINE_URL` or `VITE_PIPELINE_FRONTEND_URL`?

Reply:
I generally disagree with names that try to match what something is used _for_, preferring names that reflect what the thing actually _is_.

This is because a name can be used in multiple places and for multiple reasons. In this particular case `VITE_` is just one use case for a name that can be used for other things.

IMO this leads to confusion when using `VITE_` in the pipeline code, as in my understanding the OCurrent backend should have nothing to do with vite at all.

Reply:
I agree that it's better to not prefix these env variables with `VITE_`. I did that because:

> To prevent accidentally leaking env variables to the client, only variables prefixed with VITE_ are exposed to your Vite-processed code

See the [doc](https://vitejs.dev/guide/env-and-mode.html#env-files) for more info.

That being said and now that I think about it, we could remove the `VITE_` prefix and add it when we create the `.env` file in the [Dockerfile](https://github.com/ocurrent/current-bench/blob/main/frontend/Dockerfile#L15)

Reply:
I wasn't aware of the `VITE_` prefix requirement. I agree that redefining the var with the prefix in the Dockerfile is a good idea.

@gs0510 could you make this change, if you agree with this too?